src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBroker.cs

```diff
@@ -38,7 +38,7 @@ public AndroidBroker(CoreUIParent uiParent, ICoreLogger logger)
             _brokerHelper = new AndroidBrokerHelper(Application.Context, logger);
         }
 
-        public bool CanInvokeBroker()
+        public bool BrokerIsInstalledAndInvokable()
```

Comment:
could this be `IsBrokerInstalledAndInvokable()`?

Reply:
sure